packages/clay-css/src/scss/components/_reboot.scss

```diff
@@ -0,0 +1,477 @@
+html {
+	// Prevent adjustments of font size after orientation changes in IE on Windows Phone and in iOS
+
+	-webkit-text-size-adjust: 100%;
+
+	// Change the default tap highlight to be completely transparent in iOS
+
+	-webkit-tap-highlight-color: rgba($black, 0);
+}
+
+// Shim for "new" HTML5 structural elements to display correctly (IE10, older browsers)
+
+article,
+aside,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+main,
+nav,
+section {
+	display: block;
+}
+
+body {
+	// As a best practice, apply a default `background-color`
+
+	background-color: $body-bg;
+	color: $body-color;
+	font-family: $font-family-base;
+	font-size: $font-size-base;
+	font-weight: $font-weight-base;
+	-moz-osx-font-smoothing: $body-moz-osx-font-smoothing;
+	-webkit-font-smoothing: $body-webkit-font-smoothing;
+	line-height: $line-height-base;
+
+	// Remove the margin in all browsers
+
+	margin: 0;
+	-ms-overflow-style: scrollbar;
+	text-align: $body-text-align;
+
+	@include clay-scale-component {
+		font-size: $font-size-base-mobile;
+	}
+}
+
+hr {
+	// Add the correct box sizing in Firefox
+
+	box-sizing: content-box;
+	height: 0;
+
+	// Show the overflow in Edge and IE
+
+	overflow: visible;
+}
+
+// Remove top margins from headings
+//
+// By default, `<h1>`-`<h6>` all receive top and bottom margins. We nuke the top
+// margin for easier control within type scales as it avoids margin collapsing.
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+	margin-bottom: $headings-margin-bottom;
+	margin-top: 0;
+}
+
+// Reset margins on paragraphs
+//
+// Similarly, the top margin on `<p>`s get reset. However, we also reset the
+// bottom margin to use `rem` units instead of `em`.
+
+p {
+	margin-bottom: $paragraph-margin-bottom;
+	margin-top: 0;
+}
+
+abbr[title],
+abbr[data-original-title] {
+	// Remove the bottom border in Firefox 39-
+
+	border-bottom: 0;
+	cursor: help;
+
+	// Prevent the text-decoration to be skipped
+
+	text-decoration-skip-ink: none;
+
+	// Add the correct text decoration in Chrome, Edge, IE, Opera, and Safari
+
+	text-decoration: underline dotted;
+	text-decoration: underline;
+}
+
+address {
+	font-style: normal;
+	line-height: inherit;
+	margin-bottom: 1rem;
+}
+
+ol,
+ul,
+dl {
+	margin-bottom: 1rem;
+	margin-top: 0;
+}
+
+ol ol,
+ul ul,
+ol ul,
+ul ol {
+	margin-bottom: 0;
+}
+
+dt {
+	font-weight: $dt-font-weight;
+}
+
+dd {
+	margin-bottom: 0.5rem;
+
+	// Undo browser default
+
+	margin-left: 0;
+}
+
+blockquote {
+	margin: 0 0 1rem;
+}
+
+b,
+strong {
+	// Add the correct font weight in Chrome, Edge, and Safari
+
+	font-weight: $font-weight-bolder;
+}
+
+small {
+	// Add the correct font size in all browsers
+
+	font-size: 80%;
+}
+
+// Prevent `sub` and `sup` elements from affecting the line height in
+// all browsers.
+
+sub,
+sup {
+	font-size: 75%;
+	line-height: 0;
+	position: relative;
+	vertical-align: baseline;
+}
+
+sub {
+	bottom: -0.25em;
+}
+
+sup {
+	top: -0.5em;
+}
+
+[href] {
+	// Remove the gray background on active links in IE 10
```

Comment:
IE10?

Reply:
I wanted to remove these, but I don't have a working IE11 browser to test right now. I don't fully trust these comments.

Reply:
I just got my vm to work. I'll test and remove.